includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -529,6 +529,9 @@ public function javascript_params() {
 			$stripe_params['excludedPaymentMethodTypes']   = $this->get_excluded_payment_method_types();
 		}
 
+		// Adaptive Pricing support for checkout.
+		$stripe_params['isAdaptivePricingSupported'] = WC_Stripe_Helper::is_adaptive_pricing_supported();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'javascript_params'. (undefined)

(StaticAccess)